SpinCore_pp/ppg/generic.py

```diff
@@ -117,8 +124,10 @@ def generic(
         configureTX(adcOffset, carrierFreq_MHz, tx_phases, amplitude, nPoints)
         run_scans_time_list.append(time.time())
         run_scans_names.append("configure Rx")
-        check = configureRX(SW_kHz, nPoints, nScans, nEchoes, nPhaseSteps)
-        assert acq_time_ms == check
+        check = round(configureRX(SW_kHz, nPoints, nScans, nEchoes, int(nPhaseSteps)),1)
+        print(check)
+        print(acq_time_ms)
```

Comment:
more print statements that are diagnostic -- remove.  If you really want to check something, then use an assert